radicle-httpd/Cargo.toml

```diff
@@ -40,10 +40,12 @@ tracing-subscriber = { version = "0.3", default-features = false, features = ["s
 path = "../radicle"
 version = "0.2.0"
 
+[dependencies.radicle-cli]
+path = "../radicle-cli"
+
```

Comment:
This is a bit unfortunate tbh, maybe I can extract the coloring module out of cli so that we don't depend on the whole crate. I also think we eventually don't want to depend on it in dev-dependencies either. But it's fine for now.

Reply:
I've actually started extracting this because we need it for radicle-node too.

Reply:
> I've actually started extracting this because we need it for radicle-node too.

once done, you can ping me here and I'll update.

Reply:
btw, `tracing` uses `nu-ansi-term` which we can pull in instead, just in case.

Reply:
It's done: `radicle-term` crate.